src/main/lua/entry.lua

```diff
@@ -0,0 +1,19 @@
+---
+-- This is the main entry point of the concrete Lua Virtual Schema adapter.
+-- <p>
+-- It is responsible creating for wiring up the adapter objects.
```

Comment:
```suggestion
-- It is responsible for wiring up the adapter objects.
```

Reply:
Was meant to say "creating and wiring up".

Reply:
Fixed.